L1Trigger/L1CaloTrigger/python/Phase1L1TJets_9x9trimmed_cff.py

```diff
@@ -1,27 +1,27 @@
 import FWCore.ParameterSet.Config as cms
 from math import pi
 
-from L1Trigger.L1CaloTrigger.Phase1L1TJetProducer_cfi import Phase1L1TJetProducer
-from L1Trigger.L1CaloTrigger.Phase1L1TJetCalibrator_9x9trimmedJets_cfi import Phase1L1TJetCalibrator as Phase1L1TJetCalibrator9x9trimmed
-from L1Trigger.L1CaloTrigger.Phase1L1TJetSumsProducer_cfi import Phase1L1TJetSumsProducer
+from L1Trigger.L1CaloTrigger.Phase1L1TJetProducer_cfi import l1tPhase1JetProducer
+from L1Trigger.L1CaloTrigger.Phase1L1TJetCalibrator_9x9trimmedJets_cfi import l1tPhase1JetCalibrator as Phase1L1TJetCalibrator9x9trimmed
+from L1Trigger.L1CaloTrigger.Phase1L1TJetSumsProducer_cfi import l1tPhase1JetSumsProducer
 
-Phase1L1TJetProducer9x9trimmed = Phase1L1TJetProducer.clone(
+l1tPhase1JetProducer9x9trimmed = l1tPhase1JetProducer.clone(
 	  jetIEtaSize = 9,
 	  jetIPhiSize = 9,
 	  trimmedGrid = True,
 	  outputCollectionName = "UncalibratedPhase1L1TJetFromPfCandidates"
 )
 
-Phase1L1TJetCalibrator9x9trimmed.inputCollectionTag = ("Phase1L1TJetProducer9x9trimmed", "UncalibratedPhase1L1TJetFromPfCandidates", "")
-Phase1L1TJetCalibrator9x9trimmed.outputCollectionName = ("Phase1L1TJetFromPfCandidates")
+l1tPhase1JetCalibrator9x9trimmed.inputCollectionTag = ("l1tPhase1JetProducer9x9trimmed", "UncalibratedPhase1L1TJetFromPfCandidates", "")
```

Comment:
Why aren't you cloning also this one, instead of modifying the original config?

Reply:
Done